js/bytex.js

```diff
@@ -909,76 +1020,61 @@ module.exports = class bytex extends Exchange {
             'amount': amount,
             'filled': filled,
             'remaining': undefined,
             'cost': undefined,
             'trades': undefined,
             'fee': undefined,
-            'average': undefined,
             'info': order,
+            'average': undefined,
         }, market);
     }
 
     async createOrder (symbol, type, side, amount, price = undefined, params = {}) {
-        /**
-         * @method
-         * @name bytex#createOrder
-         * @description Create an order on the exchange
-         * @param {str} symbol Unified CCXT market symbol
-         * @param {str} type "limit" or "market" *"market" is contract only*
-         * @param {str} side "buy" or "sell"
-         * @param {float} amount the amount of currency to trade
-         * @param {float} price *ignored in "market" orders* the price at which the order is to be fullfilled at in units of the quote currency
-         * @param {dict} params  Extra parameters specific to the exchange API endpoint
-         * @param {float} params.stopPrice The price at which a trigger order is triggered at
-         * @param {dict} params.postOnly true for postOnly orders
-         * @returns [An order structure]{@link https://docs.ccxt.com/en/latest/manual.html#order-structure}
-         */
         await this.loadMarkets ();
         const market = this.market (symbol);
-        const stopPrice = this.safeFloat (params, 'stopPrice');
-        const postOnly = this.isPostOnly (type, params);
-        params = this.omit (params, [ 'stopPrice', 'postOnly', 'timeInForce' ]);
         const request = {
             'symbol': market['id'],
             'side': side,
-            'size': amount,
+            'size': this.normalizeNumberIfNeeded (amount),
             'type': type,
+            // 'stop': parseFloat (this.priceToPrecision (symbol, stopPrice)),
+            // 'meta': {}, // other options such as post_only
         };
         if (type !== 'market') {
-            if (price === undefined) {
-                throw new ArgumentsRequired (this.id + ' createOrder() requires a price argument for ' + type + ' orders');
-            }
-            request['price'] = price;
+            const convertedPrice = parseFloat (this.priceToPrecision (symbol, price));
+            request['price'] = this.normalizeNumberIfNeeded (convertedPrice);
         }
+        const stopPrice = this.safeNumber2 (params, 'stopPrice', 'stop');
         if (stopPrice !== undefined) {
-            request['stop'] = parseFloat (this.priceToPrecision (symbol, stopPrice));
-        }
-        if (postOnly) {
-            request['meta'] = {
-                'post_only': true,
-            };
+            request['stop'] = this.normalizeNumberIfNeeded (parseFloat (this.priceToPrecision (symbol, stopPrice)));
+            params = this.omit (params, [ 'stopPrice', 'stop' ]);
         }
-        const response = await this.privatePostOrder (this.deepExtend (request, params));
```

Comment:
We need to use `deepExtend` instead of `extend` because `meta` has sub properties